etc/sync-function-loader.js

```diff
@@ -1,42 +1,4 @@
 /**
- * Generates a complete sync function from the specified document definitions file.
```

Comment:
As I mentioned in the pull request description, the contents of this file have been moved verbatim to `src/sync-function-loader.js`.